release.sh

```diff
@@ -0,0 +1,172 @@
+# https://www.gnu.org/licenses/gpl-3.0.en.html) (‘GPLv3’). The programs that are governed by
+# GPLv3.0 are those programs that are located in the folders src/depends and tests/depends
+# and which include a reference to GPLv3 in their program files.
+
+# User configuration settings, mandatory to be filled in
+GitHubToken=""
+repoName="PreRelease"    # Change to Zilliqa after PreRelease is ok
+packageName=""
+releaseTitle=""
+releaseDescription=""
+
+# Environment variables
+releaseDir="release"
+versionFile="VERSION"
+majorLine=2
+minorLine=4
+fixLine=6
+DSLine=8
+commitLine=12
+shaLine=14
+sigLine=16
+
+# Validate input argument
+if [ "$#" -ne 2 ]; then
+    echo "Usage: source ./release.sh privateKeyFileName publicKeyFileName"
+    return 1
+fi
+
+if [ ! -f "$1" ]; then
+    echo "*ERROR* File : $1 not found!"
+    return 1
+fi
+
+if [ ! -f "$2" ]; then
+    echo "*ERROR* File : $2 not found!"
+    return 1
+fi
+
+if [ "$GitHubToken" = "" ]; then
+    echo "*ERROR* Please enter your own GitHub token in release.sh!"
+    return 1
+fi
+
+if [ "$packageName" = "" ]; then
+    echo "*ERROR* Please enter the package name in release.sh!"
+    return 1
+fi
+
+if [ "$releaseTitle" = "" ] || [ "$releaseDescription" = "" ]; then
+    echo "*ERROR* Please enter the release title and description in release.sh!"
+    return 1
+fi
+
+# Read current version information from version file
+defaultMajor="$(sed -n ${majorLine}p ${versionFile})"
+defaultMinor="$(sed -n ${minorLine}p ${versionFile})"
+defaultFix="$(sed -n ${fixLine}p ${versionFile})"
+defaultDS="$(sed -n ${DSLine}p ${versionFile})"
+defaultCommit="$(sed -n ${commitLine}p ${versionFile})"
+currentVer=${defaultMajor}.${defaultMinor}.${defaultFix}.${defaultDS}.${defaultCommit}
+
+# Ask user to input new version information
+echo -e "Current software version:  ${currentVer}"
+read -p "Please enter major version [${defaultMajor}]: " major
+major=${major:-${defaultMajor}}
+read -p "Please enter minor version [${defaultMinor}]: " minor
+minor=${minor:-${defaultMinor}}
+read -p "Please enter fix version [${defaultFix}]: " fix
+fix=${fix:-${defaultFix}}
+read -p "Please enter expected DS epoch [${defaultDS}]: " DS
+DS=${DS:-${defaultDS}}
+commit="$(git describe --always)"
+
+# Write new version information into version file
+echo -e "Writing new software version into ${versionFile}..."
+sed -i "${majorLine}s/.*/${major}/" ${versionFile}
+sed -i "${minorLine}s/.*/${minor}/" ${versionFile}
+sed -i "${fixLine}s/.*/${fix}/" ${versionFile}
+sed -i "${DSLine}s/.*/${DS}/" ${versionFile}
+sed -i "${commitLine}s/.*/${commit}/" ${versionFile}
+newVer=${major}.${minor}.${fix}.${DS}.${commit}
+export ZIL_VER=${newVer}
+export ZIL_PACK_NAME=${packageName}
+echo -e "New software version: ${newVer} is written into ${versionFile} successfully.\n"
+
+# Use cpack to making deb file
+echo -e "Make deb package..."
+rm -rf ${releaseDir}
+cmake -H. -B${releaseDir} -DCMAKE_INSTALL_PREFIX=..
```

Comment:
Perhaps we still need `CMAKE_BUILD_TYPE` here. Can you help find an appropriate type for the released binary?

http://cmake.org/cmake/help/v3.5/variable/CMAKE_BUILD_TYPE.html#variable:CMAKE_BUILD_TYPE

Reply:
Fine, let me add -DCMAKE_BUILD_TYPE=RelWithDebInfo back